src/main/java/xbot/common/injection/BaseWPITest.java

```diff
@@ -16,6 +16,7 @@
 @Ignore
 public class BaseWPITest {
     public Injector injector;
+    public BaseComponent injectorComponent;
```

Comment:
Should this variable maybe be `baseComponentsInjector`? It read a little backwards to me with the current word order.